src/SideNav.js

```diff
@@ -95,26 +105,16 @@ SideNav.Link = styled(Link).attrs(props => {
       text-decoration: none;
       background-color: ${get('colors.gray.1')};
       outline: none;
-
-      // Bar on the left
-      &::before {
-        background-color: ${get('colors.gray.4')};
-      }
-    }
-
-    &:active {
-      background-color: ${get('colors.white')};
```

Comment:
@auareyou in the Figma sticker sheet, for things like buttons it looks like you use the term `selected` to mean `active` (AKA the mouse is currently clicking on the item). 

For the [vertical navigation](https://www.figma.com/file/k2BopyqPZy5Wz7hAiZbnPi/Component-refresh?node-id=4031%3A228) mockup, are the `selected` state styles meant for when item in the menu is currently selected (the user had clicked on that link and is currently on that page), or does that mean the item is `active` (the mouse is currently pressed down and clicking on the item)?
![image](https://user-images.githubusercontent.com/8960591/79029114-c8cd1780-7b47-11ea-91bc-bc563ee0ea4e.png)

 I interpreted the `selected` state in the design as meaning that the item in the menu is currently selected... but should we have a special state for `active`? Right now I'm just using the hover state for active (gray background):
![image](https://user-images.githubusercontent.com/8960591/79029123-d2ef1600-7b47-11ea-9dec-3e2e98be80d5.png)

Reply:
I think your interpretation is correct. I was using selected and active as the same state without realizing selected means a different thing in components like this. 👍